dataland-frontend/src/components/general/DataPointDataTable.vue

```diff
@@ -77,7 +77,7 @@ export default defineComponent({
       const dataSource = this.dialogData.dataPointDisplay.dataSource;
       if (!dataSource) return '';
       if ('page' in dataSource) {
-        return `${dataSource.fileName}, page ${dataSource.page}`;
+        return dataSource.page === null ? `${dataSource.fileName}` : `${dataSource.fileName}, page ${dataSource.page}`;
```

Comment:
As this was a bug by our team rules we would need a short component test, that ensures that page null is no longer shown.

Reply:
test is added